pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/buffer/TopicTransactionBufferTest.java

```diff
@@ -86,4 +98,50 @@ public void testTransactionBufferAppendMarkerWriteFailState() throws Exception {
         FieldUtils.writeField(persistentTopic.getManagedLedger(), "state", ManagedLedgerImpl.State.WriteFailed, true);
         txn.commit().get();
     }
+
+    @Test
+    public void testCheckDeduplicationFailedWhenCreatePersistentTopic() throws Exception {
```

Comment:
@tjiuming The test can also get passed with the code fix. Could you please check again?

Reply:
@codelipenghui I've fixed the test, PTAL